packages/zoe/src/contractFacet/internal-types.js

```diff
@@ -28,5 +28,5 @@
  * @property {(bundle: SourceBundle) => void} evaluateContract
  * @property {(instanceAdminFromZoe: ERef<ZoeInstanceAdmin>,
       instanceRecordFromZoe: InstanceRecord,
-      issuerStorageFromZoe: IssuerRecords) => Promise<ExecuteContractResult>} startContract
+      issuerStorageFromZoe: IssuerRecords, privateArgs: Object=) => Promise<ExecuteContractResult>} startContract
  */
```

Comment:
We have to prettify jsdoc comments ourselves.

```js
 * @property {(instanceAdminFromZoe: ERef<ZoeInstanceAdmin>,
 *    instanceRecordFromZoe: InstanceRecord,
 *    issuerStorageFromZoe: IssuerRecords,
 *    privateArgs: Object=
 * ) => Promise<ExecuteContractResult>} startContract
```

Reply:
Done!